apps/docs/pages/guides/auth/social-login/auth-azure.mdx

```diff
@@ -10,63 +10,102 @@ To enable Azure (Microsoft) Auth for your project, you need to set up an Azure O
-- Select "New registration".
+- Under Microsoft Entra ID, select _App registrations_ in the side panel and select _New registration._
 - Choose a name and select your preferred option for the supported account types.
-- Specify the "Redirect URI".
-- The redirect / callback URI should look like this: `https://<project-ref>.supabase.co/auth/v1/callback`
-- Click "Register" at the bottom of the form.
+- Specify a _Web_ _Redirect URI_. It should should look like this: `https://<project-ref>.supabase.co/auth/v1/callback`
+- Finally, select _Register_ at the bottom of the screen.
 
 ![Register an application.](/docs/img/guides/auth-azure/azure-register-app.png)
 
-## Obtain a Client ID
+## Obtain a Client ID and Secret
 
-This will serve as the `client_id` when you make API calls to authenticate the user.
+- Once your app has been registered, the client ID can be found under the [list of app registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) under the column titled _Application (client) ID_.
+- You can also find it in the app overview screen.
+- Place the Client ID in the Azure configuration screen in the Supabase AUth dashboard.
 
-- Once your app has been registered, the client id can be found under the [list of app registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) under the column titled "Application (client) ID".
+![Obtain the client ID](/docs/img/guides/auth-azure/azure-client-id.png)
 
-![Obtain the client id](/docs/img/guides/auth-azure/azure-client-id.png)
+- Select _Add a certificate or secret_ in the app overview screen and open the _Client secrets_ tab.
+- Select _New client secret_ to create a new client secret.
+- Choose a preferred expiry time of the secret. Make sure you record this in your calendar days in advance so you have enough time to create a new one without suffering from any downtime.
+- Once the secret is generated place the _Value_ column (not _Secret ID_) in the Azure configuration screen in the Supabase Auth dashboard.
 
-## Obtain a Secret ID
-
-This will serve as the `client_secret` when you make API calls to authenticate the user.
+![Obtain the client secret](/docs/img/guides/auth-azure/azure-client-secret.png)
 
-- Click on the name of the app registered above.
-- Under "Essentials", click on "Client credentials".
-- Navigate to the "Client secrets" tab and select "New client secret".
-- Enter a description and choose your preferred expiry for the secret.
-- Once the secret is generated, save the `value` (not the secret ID).
+## Guarding Against Unverified Email Domains
+
+Microsoft Entra ID had a vulnerability that allowed tenants to configure any email address for their users without verification. This property could be used to gain access to already existing user accounts in Supabase Auth when a multi-tenant OAuth application was registered and Login with Azure was active.
```

Comment:
Link the post to the vulnerability here?

Reply:
It's in the resources, I wanted to avoid putting it here as it's quite jargony and not sure if people would find it super useful when reading that section.